docs/operators/rln-quickstart.md

```diff
@@ -0,0 +1,70 @@
+# Quickstart: Running a nwaku node with Rate Limiting Enabled (waku-rln-relay)
```

Comment:
Could we phrase this as a "How to" and move it into the `/how-to` subdirectory? In general I don't want a "quickstart" explosion, which should just be the essential guides to get going within specific environments (docker, droplet, general). Specific use cases or configurations fit better as how-to tutorials.

It's a good idea to make this guide as practical as possible for people. In other words, I'd prefer if we phrase the title as something like:
"Detect spam on your nwaku node using Rate Limiting Nullifiers (RLN)" (For how-to guides there is an implicit "How to" before each post title, so this would imply "How to detect spam with your nwaku node...").

Reply:
Addressed in [31b55aa](https://github.com/status-im/nwaku/pull/1311/commits/31b55aab1df804b2d6b63eb989d3e47939de14d1)